### PR TITLE
update opta.rst

### PR DESCRIPTION
**Pull request by SmritiSatyanV**
[Discussion](https://github.com/flyteorg/flyte/issues/2740)
Fixes https://github.com/flyteorg/flyte/issues/2740
Add a note about opta not being actively developed
Doc generated: https://flyte--2742.org.readthedocs.build/en/2742/deployment/overview.html#usage-of-helm
Signed-off-by: SmritiSatyanV smriti@union.ai

### CODE REVIEW

File: rsts/deployment/aws/opta.rst [hunk 1]
```diff
@@ -10,13 +10,15 @@ load robustly and securely.
 * The sandbox's object store must be replaced with a production-grade storage system
 * The sandbox's PostgreSQL database must be replaced with a production-grade deployment of PostgreSQL
```
Comment (evalsocket) [inline rsts/deployment/aws/opta.rst:11]: we also need the same note for GCP also

**Review by evalsocket — DISMISSED** [review]
LGTM

**Comment by evalsocket** [comment]
cc: @wild-endeavor do you have any comment?

File: rsts/deployment/aws/opta.rst [hunk 2]
```diff
@@ -9,14 +9,18 @@ load robustly and securely.
...
+All the above have to be done in a secure manner
+
+* (Optional) Create an official DNS domain
+* (Optional) Provision and configure a production-grade email sending system
 
 A Flyte user may provision and orchestrate this setup by themselves, but the Flyte team has partnered with the
 `Opta <https://github.com/run-x/opta>`_ team to create a streamlined production deployment strategy for AWS with
 ready-to-use templates provided in the `Flyte repo <https://github.com/flyteorg/flyte/tree/master/opta/aws>`__.
 
+.. warning:: Opta is no longer actively in development. You can `migrate to terraform  <https://docs.opta.dev/features/terraform/#migrate-from-opta-to-terraform>`__ to generate terraform code from the opta file.
```
Comment (wild-endeavor) [inline rsts/deployment/aws/opta.rst:22]: Hmm, not a fan that our recommended installation path relies on something that's no longer being maintained and has this warning.  It doesn't inspire any confidence. At the very least could we add that we're trying to create and maintain a terraform module and a timeline for when you think we'll have it?

Comment (evalsocket) [reply rsts/deployment/aws/opta.rst:22]: We can port all the steps from opta to flyte

Comment (evalsocket) [reply rsts/deployment/aws/opta.rst:22]: @SmritiSatyanV can you port the migration docs from opta to flyte?

Comment (SmritiSatyanV) [reply rsts/deployment/aws/opta.rst:22]: Sure @evalsocket

**Review by evalsocket — APPROVED** [review]

**Pull request merged**